index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for jasmine-data-provider 2.2.0
+// Project: https://github.com/MortalFlesh/jasmine-data-provider
+// Definitions by: Oleg Hoefling <https://github.com/hoefling>
+
+declare interface DataDescProvider<T> {
+  [key: string]: T;
+}
+
+declare function using<T>(data: T[], specDefinitions: (datum: T) => void): void;
```

Comment:
I'm bit confused from the `datum: T` - what does it mean? I think `item: T` or `case: T` would be more understandable..